pkg/subctl/operator/deploy/ensure.go

```diff
@@ -0,0 +1,75 @@
+package deploy
+
+import (
+	"fmt"
+
+	submariner "github.com/submariner-io/submariner-operator/pkg/apis/submariner/v1alpha1"
+	submarinerclientset "github.com/submariner-io/submariner-operator/pkg/client/clientset/versioned"
+	"github.com/submariner-io/submariner-operator/pkg/subctl/datafile"
+
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/rest"
+)
+
+func Ensure(config *rest.Config, remoteNamespace string, submarinerNamespace string,
+	clusterID string, serviceCIDR string, clusterCIDR string,
+	subctlData *datafile.SubctlData) error {
+
+	fmt.Printf("%v\n", string(subctlData.ClientToken.Data["ca.crt"]))
+
+	// Populate the Submariner CR for the operator
+	submarinerSpec := submariner.SubmarinerSpec{
+		Version:                  "",
+		Repository:               "",
+		CeIPSecNATTPort:          0,
+		CeIPSecIKEPort:           0,
+		CeIPSecDebug:             false,
+		CeIPSecPSK:               string(subctlData.IPSecPSK.Data["psk"]),
```

Comment:
I'm not sure if it will work as string, or if you need to do base64.URLEncoder.encode (or whatever the function is)